.travis.yml

```diff
@@ -5,6 +5,9 @@ php:
   - 7.3
   - 7.4
   - 8.0
+  - 8.1
+  - 8.2
+  - 8.3
 
```

Comment:
Travis is actually dead and I obviously forgot to move the test to gh actions. Would you be up to help out with making that happen?

Reply:
Add the gh actions and update PHPUnit.

I can't get the test "dumpReturnsFalseIfFileCannotBeWritten" to run on GitHub. This works well locally.

Reply:
Maybe an error with docker mounts.

Reply:
@helhum I can't get the unit test to run.
I really have no idea what it could be. I have tested vfsStream. Unfortunately this does not work with the Composer filesystem :(
Writing to the system temporary file system using`sys_get_temp_dir` didn't work either.

Do you have an idea?